pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -93,7 +98,7 @@ type Config struct {
 	CPUs                      string
 	Memory                    string
 	Namespace                 string
-	OsImage                   OSImage
+	OSImage                   *OSImage
```

Comment:
Why it has to be a pointer now?

Reply:
Notting specific, just `parseOSImage()` return pointer so I can directly assign it to `config.OSImage`